README.md

```diff
@@ -13,66 +13,65 @@ We try to increase our contribution to the iOS developer community through:
 
 # Tools We Use
+
+For employee management we use [BambooHR](https://nodesagency.bamboohr.com/home). Ask IT if ou don't have access and the Chief People Officer if anything in there is unclear. Use this to request time off, register sick day and manage your personal info and banking info for salary pay.
+
 Our main communication tool is [**Slack**](https://nodes.slack.com). We use this for daily communication.
 
-Our main project management tool is [**Trello**](https://trello.com). Each project has its own Trello board. Always keep your project's Trello board up to date to reflect the current status of the project. We work in an agile manner and move fast, but the Trello board must clearly reflect the project's status. Ask [Simon](https://nodes.slack.com/messages/@siej) or your project manager if you have questions.
+Our main project management tool is [**JIRA**](https://nodesagency.atlassian.net/jira/). Each project has its own JIRA board. Always keep your project's JIRA board up to date to reflect the current status of the project. We work in an agile manner and move fast, but the JIRA board must clearly reflect the project's status. Ask [Simon](https://nodes.slack.com/messages/@siej) or your project manager if you have questions.
 
-[**Basecamp**](https://basecamp.com) is the main tool through which we interact with the clients. As a developer, normally you shouldn't have to deal that much with the clients directly or through Basecamp, but each project is unique and in some cases, developers are also added to Basecamp.
+With the same JIRA account you will also have access to Confluence, where you can find project-related documentation, and also internal docuements.
 
-[**Harvest**](https://nodes.harvestapp.com/) is the time tracking tool that we use. It helps us see how much time one person has worked on a project. Make sure to always harvest on the appropriate project. Ask the PM on which Harvest project you should track the time. We expect you to harvest 7.5 hours per day (it's ok to do more, if you want). Lunch time is not harvested, and it doesn't count towards the 7.5 hours per day. And if you want more detailed instructions on how to harvest, ask [Simon](https://nodes.slack.com/messages/@siej).
+[**Harvest**](https://nodes.harvestapp.com/) is the time tracking tool that we use. It helps us see how much time one person has worked on a project. Make sure to always harvest on the appropriate project. Ask the PM on which Harvest project you should track the time. We expect you to harvest 7.5 hours per day. Lunch time is not harvested, and it doesn't count towards the 7.5 hours per day. And if you want more detailed instructions on how to harvest, ask [Simon](https://nodes.slack.com/messages/@siej). You can also set up auto-harvesting in you Ournodes acount and it will automatically harvest 7.5 hours per day on the project youre assigned to.
```

Comment:
This is not true in all offices, Berlin harvest 8 hours.

Reply:
Should I remove the fluffy text and just link to the Harvest section in the handbook? This way, we won't need to care about updating the playbook when/if the harvesting rules change.

Reply:
Well, I don't really mind however it's not the same across all offices :) And if this is supposed to be the global playbook it will not apply to many places.

Reply:
I updated it with an `(8 in Germany)`, removed all other mentions (I hope) of 7.5 hours and linked to the handbook instead of redirecting people to Simon :)